storage-provider/server/Cargo.toml

```diff
@@ -41,6 +41,7 @@ tokio-util = { workspace = true, features = ["rt"] }
 tower = { workspace = true }
 tower-http = { workspace = true, features = ["trace"] }
 tracing = { workspace = true }
+tracing-appender = "0.2.3"
```

Comment:
probably should be set at the workspace level

Reply:
You're right